sdk/key_vault/src/sign.rs

```diff
@@ -0,0 +1,60 @@
+use anyhow::Context;
+use azure_core::TokenCredential;
+use getset::Getters;
+use serde::Deserialize;
+use serde_json::{Map, Value};
+use url::Url;
+
+use crate::client::API_VERSION;
+use crate::key::JsonWebKeySignatureAlgorithm;
+use crate::{KeyVaultClient, KeyVaultError};
+
+#[derive(Debug, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct KeyOperationResult {
```

Comment:
`SignResult`. The algorithm should also be passed back from the `sign` call for ease of passing back to `verify`, for example (when added). See https://github.com/Azure/azure-sdk-for-java/blob/08eadff95aa5a64dcefd8ef58a5bd29d536ab736/sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/models/SignResult.java#L13